database/mysql.go

```diff
@@ -42,20 +45,33 @@ func (e *Mysql) Setup() {
 	}
 
 	// 是否开启详细日志记录
-	orm.Eloquent.LogMode(viper.GetBool("settings.gorm.logMode"))
+	// orm.Eloquent.LogMode(viper.GetBool("settings.gorm.logMode"))
+	// This decision needs an approval from the engineer - it might be that they want to change the logger type
+
+	if viper.GetBool("settings.gorm.logMode") {
+		orm.Eloquent.Logger = glogger.Default.LogMode(glogger.Info)
+	} else {
+		orm.Eloquent.Logger = glogger.Default.LogMode(glogger.Silent)
+	}
+
+	eldb, err := orm.Eloquent.DB()
+	if err != nil {
+		logger.Fatalf("database error %v", err)
```

Comment:
Initially, this line was logging the error on the GORM logger instead of adding it to the application logger. A mistake by the AI tool that was not aware.